tests/foreman/api/test_registration.py

```diff
@@ -90,7 +91,11 @@ def test_host_registration_end_to_end(
 @pytest.mark.tier3
 @pytest.mark.rhel_ver_match('[^6]')
 def test_positive_allow_reregistration_when_dmi_uuid_changed(
-    module_org, rhel_contenthost, target_sat, module_ak_with_synced_repo, module_location
+    module_entitlement_manifest_org,
+    rhel_contenthost,
+    target_sat,
```

Comment:
`module_entitlement_manifest_org`, `module_activation_key` are using module scope satellite
```suggestion
    module_target_sat,
```

Reply:
Since this test changes uuid so I wanted to keep it function scoped. I am not entirely sure but I guess the one used in test function will override the other one.

Reply:
@vijaysawant Are you good with this? We can resolve this conversation.

Reply:
@shweta83 I agree that this test should use its own satellite since it is tweaking it.